src/StateService/MPT/MPTTrie.Find.cs

```diff
@@ -62,22 +62,30 @@ private ReadOnlySpan<byte> Seek(ref MPTNode node, ReadOnlySpan<byte> path, out M
             return ReadOnlySpan<byte>.Empty;
         }
 
-        public IEnumerable<(TKey Key, TValue Value)> Find(ReadOnlySpan<byte> prefix)
+        public IEnumerable<(TKey Key, TValue Value)> Find(ReadOnlySpan<byte> prefix, byte[] from = null)
```

Comment:
If from key is specified, it should not be included in the results returned by Find.

Reply:
Note, this is still an issue as of latest commit in this PR (661284c08907608a93c7331fd90084de89af27a1)

Reply:
I have not finished this fix. Will inform you then.

Reply:
Done.